src/system_metrics/service_mongoose_system_metrics.erl

```diff
@@ -8,6 +8,14 @@
 -define(DEFAULT_REPORT_AFTER, timer:hours(3)).
 -define(TRACKING_ID, "UA-151671255-2").
 -define(TRACKING_ID_CI, "UA-151671255-1").
+-define(MSG_REMOVED_FROM_CONFIG,
+        "Are you sure you don't want to report anything?"
+        "TODO: Explain this better"
```

Comment:
Sth around this (that's only for insporation):
"We need them only to improve the product. Please take a look {{here}} and/or {{here}} to understand what metrics are we interested in. In order to disable reporting completely and silance the long you can configure service_mongoose_system_metrics with that or that options"